tests/test_validate.py

```diff
@@ -525,9 +526,6 @@ def test_use_own_schema_invalid_json(self) -> None:
         )
 
 
-list_of_spec_versions_containing_licensing = ["1.5"]
-
-
 class TestValidateUseSchema15(unittest.TestCase):
```

Comment:
@CBeck-96 you have this specific class for `1.5`, but I think they also apply for `1.6` as you are just testing `licensing` which was introduced with `1.5` and still is valid for `1.6`?

Reply:
Renamed the Class to reflect its purpose and removed the test that did not align with it